modules/rasBidAdapter.md

```diff
@@ -0,0 +1,51 @@
+# Overview
+
+```
+Module Name: Ringier Axel Springer Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: support@ringpublishing.com
+```
+
+# Description
+
+Module that connects to Ringer Axel Springer demand sources.
+Only banner format is supported.
+
+# Test Parameters
+```js
+var adUnits = [{
+  code: 'test-div-ad',
+  mediaTypes: {
+    banner: {
+      sizes: [[300, 250], [300, 600]]
+    }
+  },
+  bids: [{
+    bidder: 'ras',
+    params: {
+      network: '4178463',
+      site: 'test',
+      area: 'areatest',
+      slot: 'slot'
+    }
+  }]
+}];
+```
+
+# Parameters
+
+| Name | Scope | Type | Description | Example
+| --- | --- | --- | --- | ---
+| network | required | String | Specific identifier provided by RAS | `"4178463"`
+| site | required | String | Specific identifier name (case-insensitive) that is associated with this ad unit and provided by RAS | `"example_com"`
+| area | required | String | Ad unit category name; only case-insensitive alphanumeric with underscores and hyphens are allowed | `"sport"`
+| slot | required | String | Ad unit placement name (case-insensitive) provided by RAS | `"slot"`
+| pageContext | optional | Object | Web page context data | `{}`
+| pageContext.dr | optional | String | Document referrer URL address | `"https://example.com/"`
+| pageContext.du | optional | String | Document URL address | `"https://example.com/sport/football/article.html?id=932016a5-02fc-4d5c-b643-fafc2f270f06"`
```

Comment:
I don't see the `pageContext.dv` property being used. Is this still needed?

Reply:
You are right: `pageContext.du` wasn't used but it should. Fixed